learn/getting-started/working-with-data/using-query-expressions-to-manipulate-data.md

```diff
@@ -0,0 +1,250 @@
+---
+layout: using-query-expressions-to-manipulate-data
+title: Using Query Expressions to Manipulate Data
+permalink: /learn/using-query-expressions-to-manipulate-data
+description: This simple guide helps you understand how query expressions can be used on data to produce new data 
+active: using-query-expressions-to-manipulate-data
+intro: This simple guide helps you understand how query expressions can be used on data to produce new data 
+redirect_from:
+  - /learn/using-query-expressions-to-manipulate-data
+---
+
+This simple guide helps you understand how query expressions (a.k.a language integrated queries) can be used on data to produce new data.
+
+Ballerina supports first-class support for writing SQL-like queries to process data.
+
+Language integrated queries can process any ballerina iterable collection.
+
+In this tutorial, you will be writing queries to filter, sort and join with other data sets and produce new data sets.
+
+This tutorial includes the following steps.
+
+1. Create a new Ballerina project
+2. Define the Covid dataset to be processed
+3. Find all countries which have more than 10,000,000 Covid cases
+4. Find the top three countries by the number of reported Covid cases
+5. Join the table with an array to find the number of recovered patients of three random countries
+6. Find any discrepancies in reported Covid dataset while maintaining intermediate states
+
+## Prerequisite
+
+Following two prerequisites are needed for this tutorial. 
+
+1. Install [Ballerina Swan Lake](https://ballerina.io/downloads/)
+2. Install [VSCode](https://code.visualstudio.com/download) with [Ballerina extension](https://marketplace.visualstudio.com/items?itemName=WSO2.Ballerina)
+
+
+## Create a new Ballerina project
+
+To create a new Ballerina package, use the `bal new` command as shown below.
```

Comment:
```suggestion
To create a new Ballerina package, use the `bal new` command as shown below.
```
See the review comments in the GraphQl guide PR https://github.com/ballerina-platform/ballerina-dev-website/pull/3369.